runtime/v2/bundle.go

```diff
@@ -115,16 +117,24 @@ type Bundle struct {
 }
 
 // Delete a bundle atomically
-func (b *Bundle) Delete() error {
+func (b *Bundle) Delete(ctx context.Context) error {
 	work, werr := os.Readlink(filepath.Join(b.Path, "work"))
 	rootfs := filepath.Join(b.Path, "rootfs")
+
+	// on windows hcsshim writes panic logs in the bundle directory in a file named
+	// "panic.log" log those messages (if any).
+	_, err := cioutil.ReadFileInChunks(filepath.Join(b.Path, "panic.log"), 4096, log.G(ctx).Logger.Out)
```

Comment:
The thing that I don't like about this is we need to pass the logrus writer like log.L.Logger.Out which doesn't allow us to log the panics with a message like "shim panic logs found" so searching for panic logs in kusto can get a little cumbersome. But panic logs still start with "panic" keyword so I guess it shouldn't be too bad. What do you guys think?

Reply:
I was suggesting we be defensive about how much we read and log rather than just reading in chunks. For instance read and log the first x MB of the panic file plus an error if the file is larger than that (so we can determine if we are routinely missing data). I don't know a reasonable size that would hold most panic logs though.

Reply:
The panic stacks only contains the stack from the goroutine it was encountered in right (unless you screw with the GOTRACEBACK env var)? I'd be surprised if it was over a MB

Reply:
So maybe 1MB or less. We can just allocate, open, read and for extra credit log if there was more data.

Reply:
That sounds good!

Reply:
Fixed. PTAL.